src/utils/getMonthName.ts

```diff
@@ -0,0 +1,44 @@
+import { WEEK_LENGTH } from '../constants';
+import { FullMonthNames } from '../types';
+
+const date = new Date();
+
+export default function getMonthName(
+  monthOffset = 0,
+  weekOffset = 0,
+  month = date.getMonth()
+) {
+  if (monthOffset !== 0 && weekOffset === 0) {
+    month = new Date(
+      date.getFullYear(),
+      date.getMonth() + monthOffset
+    ).getMonth();
+  } else if (weekOffset !== 0 && monthOffset === 0) {
+    month = new Date(
+      date.getFullYear(),
+      date.getMonth(),
+      date.getDate() + weekOffset * WEEK_LENGTH
+    ).getMonth();
+  } else if (weekOffset && monthOffset) {
+    month = new Date(
+      date.getFullYear(),
+      date.getMonth() + monthOffset,
+      date.getDate() + weekOffset * WEEK_LENGTH
+    ).getMonth();
```

Comment:
Maybe these 3 if statements are not necessary? Similarly as to previous comment, if monthOffset is 0, then  `date.getMonth()` or `date.getMonth() + monthOffset` produces the same result. Also same with weekOffset